### DISCUSSION TITLE
Github Training Manual: Missing file "##-USERNAME.md"

### DISCUSSION THREAD

**Discussion by thebitsflipper** [community/community#53220] [New to GitHub]
Hello,

I'm trying to follow Github Training Manual - URL:
https://githubtraining.github.io/training-manual/#/01_getting_ready_for_class

In the tutorial, I reached the following page:
https://githubtraining.github.io/training-manual/#/06_working_locally

That page has some steps under the title "Activity: Edit Your File".
One of the steps tells the student to find a file named "##-USERNAME.md" or something similar.

But I cannot find anything like that.
Here's what I have in the cloned local repository of the fork:

```
$ git branch -a
* github-thebitsflipper-caption
  master
  remotes/origin/HEAD -> origin/master
  remotes/origin/github-thebitsflipper-caption
  remotes/origin/master
$ ls -1a
.
..
.git
README.md
$ cat README.md 
# A scratch repo on GitHub.com
$ git remote show origin
* remote origin
  Fetch URL: https://github.com/thebitsflipper/scratch.git
  Push  URL: https://github.com/thebitsflipper/scratch.git
  HEAD branch: master
  Remote branches:
    github-thebitsflipper-caption tracked
    master                        tracked
  Local branches configured for 'git pull':
    github-thebitsflipper-caption merges with remote github-thebitsflipper-caption
    master                        merges with remote master
  Local refs configured for 'git push':
    github-thebitsflipper-caption pushes to github-thebitsflipper-caption (up to date)
    master                        pushes to master                        (up to date)
```

**Comment by a-a-ron** [comment]
Hi @thebitsflipper, thanks for reaching out! 

The section you are referring to is part of an interactive session designed to be used as instructor led training. The `##-USERNAME.md` files for each contributor of the class repo are generated using the class scripts. Here is a link to the training scripts used in the manual: https://githubtraining.github.io/training-manual/#/app_teaching_script

The reference to these scripts is at the end of the manual in the **Teaching Resources** section so can be easily missed. I'll see if we can move this information to the **Getting Started** section or at least mention the usage of these scripts at the beginning of the manual. These scripts are used to create the initial class repository and for generating files for projects 1 and 3, and for creating conflicts used for project 2.

**Comment by thebitsflipper** [comment]
Hi @a-a-ron ,

I understand, then it seems necessary to have some awareness to these scripts at the beginning of the manual.

I had read the page entitled "GitHub training scripts" in this URL:
https://thebitsflipper.github.io/training-manual/#/app_teaching_script?id=github-training-scripts

Though still something isn't completely clear, here are the details:

1. It is written:
> Note that these scripts are meant to interact with an organization on GitHub.com

Would it mean that I shouldn't execute these scripts if I'm not associated with any organization?

2. It is written:
> Follow the directions for setup and use located in the /script [README.md](https://github.com/githubtraining/training-manual/tree/main/script#readme) file

The link "README.md" has this URL:
https://github.com/githubtraining/training-manual/tree/main/script#readme

The contents on the page is unclear either:
2.1. Under title "First Time Usage" immediately there is the title "Remote Configuration", which has the following text:
> To use these scripts, please follow these steps first:
>
>    Create an organization that will house all student repositories. Something like "githubtraining" or "githubschool" will work, but any name will work.

How do I know whether I am supposed to execute it?
What does "Remote Configuration" even mean? Does it mean to an "Enterprise"?

2.2. Under title "Local Configuration" it is written:
> run script/teach-class and choose option 0 to set up your environment variables. You'll need the following pieces of information from [above](https://github.com/githubtraining/training-manual/tree/main/script#remote-configuration):
>
>    the title of your newly created training organization
>    your username, or the username of a shared teaching account

Similarly, it seems like these instructions are for teaching instead of just learning, so it is unclear why I should execute it.

I appreciate your help.
